CHANGELOG.md

```diff
@@ -40,6 +40,11 @@
   keyword when it can be omitted inside closures.  
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#1194](https://github.com/realm/SwiftLint/issues/1194)
+  
+* Add option to `unused_optional_binding` rule to ignore `try?`
```

Comment:
Since this change causes `try?` to be ignored _by _default, it seems to make sense to reverse the logic in this sentence:

> Add `ignore_optional_try ` option to `unused_optional_binding` rule, `true` by default. Set to `false` to emit a violation when binding the result of `try?` to `_`.